build_tools/aws-sdk-code-generator/spec/interfaces/client/idempotency_tokens_spec.rb

```diff
@@ -25,7 +25,6 @@
     end
 
     it 'does not modify given idempotency tokens' do
-      expect(SecureRandom).not_to receive(:uuid)
```

Comment:
invocation id now gets a random id, but we test below that the client_token is preserved, so this can be removed without weakning the tst.